src/mscorlib/src/System/ReadOnlySpan.cs

```diff
@@ -260,6 +312,21 @@ public bool TryCopyTo(Span<T> destination)
             SpanHelper.CopyTo<T>(ref destination.DangerousGetPinnableReference(), ref DangerousGetPinnableReference(), _length);
             return true;
         }
+
+        /// <summary>
+        /// Returns true if left and right point at the same memory and have the same length.  Note that
+        /// this does *not* check to see if the *contents* are equal.
+        /// </summary>
+        public static bool operator ==(ReadOnlySpan<T> left, ReadOnlySpan<T> right)
+        {
```

Comment:
Could you please update the `Span<T>` implementation to match, and delete `Equals(Span<T>)` ?